File: Python/Codewars/7/vowelCount.py
import re


def get_count(input_str):
    num_vowels = 0
    # your code here
    num_vowels = len(re.findall(r"[aeiou]", input_str))
    return num_vowels

# example de flag  re.findall(r"[aeiou]", input_str, re.I | )
# still need to train with using regex with python
#  pattern = re.compile(r"[aeiou]", re.I)
#         matches = pattern.finditer(string)


def getCount2(inputStr):
    return sum(1 for let in inputStr if let in "aeiouAEIOU")


def getCount3(s):
    return sum(c in 'aeiou' for c in s)

File: Javascript/Codewars/7/vowelCount.js
function getCount(str) {
  return (str.match(/[aeiou]/gi) || []).length;
}

function getCount2(str) {
  var vowelsCount = 0;
  const vowels = ["a", "e", "i", "o", "u"];
  // enter your majic here
  for (let i = 0; i < str.length; i++) {
    for (let j = 0; j < vowels.length; j++) {
      if (str[i] === vowels[j]) {
        vowelsCount++;
      }
    }
  }
  return vowelsCount;
}

function getCount(str) {
  return str.split("").filter((c) => "aeiouAEIOU".includes(c)).length;
}

//getCount2 maitrisé;
